REQUIRE

```diff
@@ -2,3 +2,4 @@ julia 0.4
 TaylorSeries 0.3.0
 FactCheck 0.4.2
 Compat 0.9.1
+ValidatedNumerics 0.5.0
```

Comment:
`ValidatedNumerics.jl` is used in the notebooks (which does not need require, and in the tests. If it is not used elsewhere, I suggest to create `tests/REQUIRE`, and include this line there.